Loader/Config/Settings.lua

```diff
@@ -284,9 +284,26 @@ settings.CodeExecution = false			-- Enables the use of code execution in Adonis;
 settings.SilentCommandDenials = false	-- If true, there will be no differences between the error messages shown when a user enters an invalid command and when they have insufficient permissions for the command
 settings.OverrideChatCallbacks = true		-- If the TextChatService ShouldDeliverCallbacks of all channels are overridden by Adonis on load. Required for slowmode. Mutes use a CanSend method to mute when this is set to false.
 
+--[[ Messages shown to banned users
+	The different placeholders you can use are:
+	{reason} - Shows the Ban Reason, if none is provided, it will show as "No Reason Provided"
+	{time} - Shows the Time they were banned, in UTC time
+	{name} - Shows the Player's name
+	{id} - Shows the Player's UserId
+	{moderator} - Shows the moderator who banned the Player
+	{type} - Shows the type of ban, Can display: "TIMEBAN", "SERVERBAN", "GAMEBAN", "CONFIGBAN", "TRELLOBAN"
+	{expiretime} - Shows the time a ban will expire, only applicable on a Timeban, else will return "Undefined" or "Permanent"
+	{remainingtime} - Shows the remaining time on a Timeban, any other form of ban will return "Undefined" or "Permanent"
+--]]
+	
 settings.BanMessage = "Banned"				-- Message shown to banned users upon kick
```

Comment:
Change this to `You have been banned from this server for {reason}.`

Reply:
Idk about changing an iconic feature of Adonis

Reply:
Is it really that iconic?

Reply:
obv